app/models/letter.rb

```diff
@@ -29,5 +29,23 @@ def letter_date
   def template_name
     @letter_template&.name
   end
+
+  def letter_recipient
```

Comment:
There is a tiny bit of duplicated logic here - I think it's fine but wonder if we could avoid this with something like a LetterRecipient class.

I don't think this is a blocker just a thought as I saw similar shapes in the code.

Reply:
tbh though I think this implementation is fine and not overly complicated 👍

Reply:
I like the idea :)